Please get [failure logs here](https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/95182/artifacts?artifactName=failure%20logs%20for%20x64-windows&api-version=7.1&%24format=zip).
```
CMake Error at D:/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/share/cmake-3.27/Modules/WriteBasicConfigVersionFile.cmake:43 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/share/cmake-3.27/Modules/CMakePackageConfigHelpers.cmake:239 (write_basic_config_version_file)
  CMakeLists.txt:31 (write_basic_package_version_file)
```

--- a/ports/murmur3/CMakeLists.txt
+++ b/ports/murmur3/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.19)
-project(murmur3 LANGUAGES C)
+project(murmur3 VERSION 1430524800 LANGUAGES C)
 
 set(Header_Files "${PROJECT_NAME}.h")
 set(Source_Files "${PROJECT_NAME}.c")

--- a/ports/murmur3/CMakeLists.txt
+++ b/ports/murmur3/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.19)
-project(murmur3 VERSION 1430524800 LANGUAGES C)
+project(murmur3 VERSION 2015.05.02 LANGUAGES C)
 
 set(Header_Files "${PROJECT_NAME}.h")
 set(Source_Files "${PROJECT_NAME}.c")

--- a/ports/murmur3/portfile.cmake
+++ b/ports/murmur3/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO            PeterScott/${PORT}
+    REPO            PeterScott/murmur3
     REF             dae94be0c0f54a399d23ea6cbe54bca5a4e93ce4
     SHA512          1bc01eefc04f06704800a7448231db9f82fc809079bd3f43ef24d7dd3d8deaec2143f252a8e556dafe366401f898b676922b0c93ac181aaf38ae69ad638adbba
     HEAD_REF        master
@@ -19,11 +19,7 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(PACKAGE_NAME unofficial-${PORT})
 
 file(WRITE "${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright" "CC0-1.0")
-file(WRITE "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage" [=[
-${PORT} provides CMake targets:
-    find_package(unofficial-${PORT} CONFIG REQUIRED)
-    target_link_libraries(main PRIVATE unofficial::${PORT}::${PORT})
-]=])
-
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/usage"
+     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share"
                     "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/murmur3/portfile.cmake
+++ b/ports/murmur3/portfile.cmake
@@ -19,7 +19,8 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(PACKAGE_NAME unofficial-${PORT})
 
 file(WRITE "${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright" "CC0-1.0")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/usage"
-     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage")
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage"
+     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share"
                     "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/murmur3/usage
+++ b/ports/murmur3/usage
@@ -1,0 +1,3 @@
+murmur3 provides CMake targets:
+    find_package(unofficial-murmur3 CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE unofficial::murmur3::murmur3)

--- a/versions/m-/murmur3.json
+++ b/versions/m-/murmur3.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f5d312dcf0dd9300043d9d6f3911cb86c225e0dc",
+      "git-tree": "2f774b401cfb3d3ba3931a9c6d854f6981787c8c",
       "version-date": "2015-05-02",
       "port-version": 0
     }

--- a/versions/m-/murmur3.json
+++ b/versions/m-/murmur3.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "0366e11399ebd483ed2cb65b98d2785f6a084c7e",
+      "git-tree": "4a1af63837424ece7a5f1377e214e6737be8d7e0",
       "version-date": "2015-05-02",
       "port-version": 0
     }

--- a/versions/m-/murmur3.json
+++ b/versions/m-/murmur3.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "4a1af63837424ece7a5f1377e214e6737be8d7e0",
+      "git-tree": "f5d312dcf0dd9300043d9d6f3911cb86c225e0dc",
       "version-date": "2015-05-02",
       "port-version": 0
     }

